content/en/building/guides/debugging/secure-sharing-of-developer-instance.md

```diff
@@ -80,7 +80,7 @@ The steps in this guide can be done on any of the cheap server providers out the
 
 From the SSH command in step 10 above:
 
-{{< figure src="../images/ssh.ports.svg" link="../images/ssh.ports.svg" caption="Explanation of ports and IPs from SSH" >}}
+![SSH.tunnel.diagram2.svg.png](images/SSH.tunnel.diagram2.svg.png)
```

Comment:
OMG - I don't know why this image is giving me so much trouble!!! It works as is so I'm not fixing it (ie make it a nice figure with an alt text)